components/CreatePost.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
+
+type NewPost = {
+  titleDescription: string;
+  type: string;
+};
+
+const iniState = {
+  titleDescription: "",
+  type: "Life",
+};
+
+export const CreatePost = () => {
+  const POST_TYPES = ["Life", "Job", "Joke", "Animals", "Food"];
+  const [isHidden, setIsHidden] = useState(true);
```

Comment:
should be more descriptive. what is hidden? you can use
`isCreatePostModalHidden` or `isPostModalHidden` something like that